ui/app/components/transaction-list-item.js

```diff
@@ -39,6 +40,8 @@ TransactionListItem.prototype.render = function () {
     txParams = transaction.msgParams
   }
 
+  const nonce = (new BN(txParams.nonce.substr(2))).toString(10)
```

Comment:
web3.currentProvider.sendAsync([ id: 1, method: 'eth_sendTransaction', params: [{
  from: web3.eth.accounts[0],
  to: web3.eth.accounts[0],
  nonce: 10
}])